proto/viam/app/v1/app.proto

```diff
@@ -951,6 +953,15 @@ message ListRegistryItemsResponse {
   repeated RegistryItem items = 1;
 }
 
+message DeleteRegistryItemRequest {
+  // The id of the item (formatted as prefix:name where prefix is the owner's orgid or namespace)
```

Comment:
Is this true? I mirrored `module_id`

Reply:
Yup! that is correct.

Reply:
technically package type is unnecessary because orgID <> name are unique. but i'm fine leaving this in just for protection. i imagine the get registry item call will use the same request shape?

Reply:
Great point. I was out of sync with that decision and this tech spec. I think lets remove type then, less to cleanup later. I can re-approve to regenerate the code after the proto is updated.